### PR TITLE
mgr/dashboard: smart: add support for NVMe

### PR DESCRIPTION
**Pull request by p-se**
Before this PR the front-end received different data than expected when SMART data of an NVMe was returned and failed silently in the browsers console, leaving the front-end unusable (until reloaded). This PR add support for the SMART data structure returned by `smartmontools` > 7.

Fixes: https://tracker.ceph.com/issues/42483

Signed-off-by: Patrick Seidensal <pseidensal@suse.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by p-se** [comment]
jenkins test make check

**Comment by bk201** [comment]
jenkins test dashboard

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/smart-list/smart-list.component.html [hunk 1]
```diff
@@ -37,9 +39,17 @@
 
             <tab i18n-heading
                  heading="S.M.A.R.T">
```
Comment (bk201) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/smart-list/smart-list.component.html]: For consistency, should we also reword this header from `S.M.A.R.T` to `SMART`?

File: src/pybind/mgr/dashboard/services/ceph_service.py [hunk 2]
```diff
@@ -173,27 +172,27 @@ def send_command(cls, srv_type, prefix, srv_spec='', **kwargs):
             raise SendCommandError(outs, prefix, argdict, r)
 
         try:
-            return json.loads(outb)
+            return json.loads(outb or outs)
```
Comment (p-se) [inline src/pybind/mgr/dashboard/services/ceph_service.py]: outs is not required anymore

**Comment by callithea** [comment]
- [pkg build](https://shaman.ceph.com/builds/ceph/wip-laura-testing-32708/) [WIP]

**Pull request merged**